### PR TITLE
Extend product schema with geotagging. Include GoogleMaps as show case.

### PR DESCRIPTION
**Pull request by prinzdezibel**
Extend simple product schema with lat and lng

Use these in a new component to display a Google Map.

### CODE REVIEW

**Comment by brent-hoover** [comment]
@prinzdezibel Nice!

**Comment by prinzdezibel** [comment]
@zenweasel Should be ok now. Google Map Api key is now sourced from `dev.settings.json`

File: client/components/availabilityMap.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        });
+
+        // eslint-disable-next-line no-undef, no-new
+        new google.maps.Marker({
+          position: {
+            lat: this.props.product.lat,
+            lng: this.props.product.lng
+          },
+          map: map,
+          title: "Buy here!"
```
Comment (brent-hoover) [inline client/components/availabilityMap.js]: Should have i18n

Comment (brent-hoover) [reply client/components/availabilityMap.js]: Probably should shy away from using JS keywords (e.g. `map`) as variable names

File: client/styles/layout.less [hunk 2]
```diff
@@ -14,3 +14,8 @@
 .bkdebug em {
   font-weight: 700;
 }
+
+.map {
+  height: 150px;
+  width: 100%;
+}
```
Comment (brent-hoover) [inline client/styles/layout.less]: missing newline

File: client/templates/layouts/core.js [hunk 3]
```diff
@@ -12,45 +14,68 @@ class CoreLayoutBeesknees extends Component {
...
-        { Template[layoutFooter] &&
-          <Blaze template={layoutFooter} className="reaction-navigation-footer footer-default" />
-        }
+
+        {headerComponent && React.createElement(headerComponent, {})}
+
+        <Blaze template="cartDrawer" className="reaction-cart-drawer"/>
+
+        <main>
+          <div className="rui beesknees">
```
Comment (brent-hoover) [inline client/templates/layouts/core.js]: Do you think this debug header is still relevant? Is there a way we could give it more value or should we just remove it?

**Review by brent-hoover — CHANGES_REQUESTED** [review]
Some style stuff and questions?

File: client/templates/layouts/core.js (hunk 3 above)
Comment (prinzdezibel) [reply client/templates/layouts/core.js]: @zenweasel  Can you elaborate a bit? Don't know what  `debug header` is exactly.

Comment (prinzdezibel) [reply client/templates/layouts/core.js]: I'd leave that as is. I find it useful as , well, debug bar..

File: client/components/availabilityMap.js (hunk 1 above)
Comment (prinzdezibel) [reply client/components/availabilityMap.js]: pulled in i18n for plugin.

File: client/styles/layout.less (hunk 2 above)
Comment (prinzdezibel) [reply client/styles/layout.less]: done.

**Comment by brent-hoover** [comment]
@prinzdezibel Just need to resolve conflicts and this should be ready to go

**Comment by prinzdezibel** [comment]
@zenweasel we should be good to go.

**Review by brent-hoover — APPROVED** [review]

**Pull request merged**